frontend/__tests__/Contact.test.jsx

```
@@ -144,7 +144,75 @@ describe('Contact Component', () => {
+  };
+  
+  beforeEach(() => {
+    vi.resetAllMocks();
+    mockNavigate.mockClear();
+    mockPreventDefault.mockClear();
+    useNavigate.mockReturnValue(mockNavigate);
+    vi.stubGlobal('import.meta', { env: { VITE_API_URL: 'https://api.example.com' } });
+    vi.spyOn(console, 'log').mockImplementation(() => {});
+    vi.spyOn(console, 'error').mockImplementation(() => {});
+    vi.spyOn(window, 'alert').mockImplementation(() => {});
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)